### ISSUE TITLE
Problem with step_3_generate_media cbind

### ISSUE THREAD

**Issue by dmacoritto** [facebookexperimental/siMMMulator#9]
I have run into issues when running this function with channel which have cpm and other which have cpc.

When running this function, I get an error: `Error in data.frame(..., check.names = FALSE) : arguments imply differing number of rows`.
I have been digging into the function and I believe the error comes from binding the rows, it comes when running this operation: `cbind(df_ads_step2, true_cpm, noisy_cpm, true_cpc, noisy_cpc)`.  `true_cpm` and `true_cpc` are actually shorter dataframes than the rest.

Could you please have a look ?

Thanks

**Comment by njessica98** [comment]
Hi! What did you use as the function's inputs? You'll have to specify CPM and CPC for ALL channels. If they don't use impressions or clicks, you'll have to put 'NA' in its place. Here is some example code: 

`df_ads_step3 <- step_3_generate_media(
  my_variables = my_variables,
  df_ads_step2 = df_ads_step2,
  true_cpm = c(2, 20, NA),
  true_cpc = c(NA, NA, 0.25),
  mean_noisy_cpm_cpc = c(1, 0.05, 0.01),
  std_noisy_cpm_cpc = c(0.01, 0.15, 0.01)
)
`
There are 3 channels here, Facebook; TV; Search. Facebook and TV use impressions and so I've specified their true CPMs. Search does not use impressions so I left the third CPM as NA. I did the same analog for CPC. 

Could you try this and let me know if it still doesn't work?

**Comment by dmacoritto** [comment]
Yes, it works. I did not understand I needed to put NAs. I was putting nothing instead and that was creating the error.
